integration-tests/tests/src/index.ts

```diff
@@ -29,16 +29,16 @@ import "./setup-globals";
 
 afterEach(() => {
   // Trigger garbage collection after every test, if exposed by the environment.
-  if (typeof global.gc === "function") {
-    global.gc();
+  if (typeof gc === "function") {
+    gc();
```

Comment:
This _might_ be causing the crash 🤔 Perhaps this didn't work as expected, before the change.